addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/handler/TrackerHandler.java

```diff
@@ -272,29 +277,27 @@ private void triggerRegionChannel(@NonNull String regionName, @NonNull String ev
      * @param message Message.
      */
     private void updateChannelsWithLocation(LocationMessage message) {
-        updateBaseChannels(message, CHANNEL_BATTERY_LEVEL, CHANNEL_LAST_LOCATION, CHANNEL_LAST_REPORT);
+        updateBaseChannels(message, CHANNEL_BATTERY_LEVEL, CHANNEL_LAST_LOCATION, CHANNEL_LAST_REPORT, CHANNEL_GPS_ACCURACY);
 
         String trackerId = message.getTrackerId();
         logger.debug("Updating distance channels tracker {}", trackerId);
         distanceChannelMap.values()
-                .forEach(c -> {
-                    updateDistanceChannelFromMessage(message, c);
-                });
+                .forEach(c -> updateDistanceChannelFromMessage(message, c));
     }
 
     private void updateDistanceChannelFromMessage(LocationMessage message, Channel c) {
+        String regionName = ConfigHelper.getRegionName(c.getConfiguration());
         PointType center = ConfigHelper.getRegionCenterLocation(c.getConfiguration());
-        PointType newLocation = message.getTrackerLocation();
+         PointType newLocation = message.getTrackerLocation();
         if (center != null) {
             double newDistance = newLocation.distanceFrom(center).doubleValue();
             updateState(c.getUID(), new QuantityType<>(newDistance / 1000, MetricPrefix.KILO(SIUnits.METRE)));
-            logger.trace("Region center distance from tracker location {} is {}m", newLocation.toString(), newDistance);
+            logger.trace("Region {} center distance from tracker location {} is {}m", regionName, newLocation.toString(), newDistance);
```

Comment:
Remove `toString()`. The logger takes care of that.

Reply:
Ok.